airrohr-firmware/platformio.ini

```diff
@@ -44,10 +44,12 @@ lib_deps_generic_external =
   Adafruit SHT31 Library@1.1.6
   ArduinoJson@6.13.0
   DallasTemperature@3.8.0
-  ESP8266_SSD1306@4.1.0
+  thingpulse/ESP8266 and ESP32 OLED driver for SSD1306 displays @ 4.1.0 ; updated SSD1306 url
   1655@1.0.2 ; TinyGPSPlus, formerly this was referenced as mikalhart/TinyGPSPlus#v0.95
   797@1.0.1 ; TinyWireM
-
+  https://github.com/CodeForAfrica/sensors.AFRICA-Adafruit_FONA.git ; sensors.AFRICA FONA library 
+  
```

Comment:
@gideonmaina , let's not add unnecessary white spaces

Reply:
Okay @Warukira